docs/releases/1.11.txt

```diff
@@ -122,7 +122,9 @@ Email
 File Storage
 ~~~~~~~~~~~~
 
-* ...
+* :class:`~django.core.files.File` now has the ``readable()``, ``writable()``
```

Comment:
To make it wrappable by TextIOWrapper, File now has the readable(), writable(), and seekable() methods.